app/views/client/displays_tab.php

```diff
@@ -1,99 +1,148 @@
-					WHERE serial_number = '$serial_number'
-					ORDER BY type";
-?>
 
-	<h2>Displays</h2>
-		<?php foreach($display->query($sql) as $obj): ?>
-			<span class="label label-success nw-displayscount"></span>
-			<table class="table table-striped">
-				<thead>
-					<tr>
-						<th>
-							<?php
-								switch ($obj->vendor) {
-									case "610":
-										echo "Apple";
-										break;
-									case "10ac":
-										echo "Dell";
-										break;
-									case "5c23":
-										echo "Wacom";
-										break;
-									case "4d10":
-										echo "Sharp";
-										break;
-									case "1e6d":
-										echo "LG";
-										break;
-									case "38a3":
-										echo "NEC";
-										break;
-									case "4c49":
-										echo "SMART Technologies";
-										break;
-									case "9d1":
-										echo "BenQ";
-										break;
-									case "4dd9":
-										echo "Sony";
-										break;
-									case "472":
-										echo "Acer";
-										break;
-									case "22f0":
-										echo "HP";
-										break;
-									case "34ac":
-										echo "Mitsubishi";
-										break;
-									case "5a63":
-										echo "ViewSonic";
-										break;
-									case "4c2d":
-										echo "Samsung";
-										break;
-									case "593a":
-										echo "Vizio";
-										break;
-									case "d82":
-										echo "CompuLab";
-										break;
-								}
-							?>
-							<?php echo $obj->model; ?> (<?php echo ($obj->type == 0) ? 'Built-in' : 'External'; ?>)
-						</th>
-						<th></th>
-					</tr>
-				</thead>
-				<tbody>
-					<tr>
-						<td>Serial</td>
-						<td><?php echo $obj->display_serial; ?></td>
-					</tr>
-					<tr>
-						<td>Manufacture date</td>
-						<td><?php echo $obj->manufactured; ?></td>
-					</tr>
-					<tr>
-						<td>Native resolution</td>
-						<td><?php echo $obj->native; ?></td>
-					</tr>
-					<tr>
-						<td>Was connected</td>
-						<td><time title="<?php echo strftime('%c',$obj->timestamp); ?>" datetime="<?php echo date('c',$obj->timestamp); ?>"></time></td>
-					</tr>
-				</tbody>
-			</table>
-		<?php endforeach; ?>
+<div id="displays-tab"></div>
+<h2 data-i18n="displays.displays"></h2>
 
-<script type="text/javascript" charset="utf-8">
-	// Set displays count in tab header
-	$(document).ready(function() {
-		$('#displays-cnt').html($('.nw-displayscount').length)
-	} );
+<script>
+$(document).on('appReady', function(){
+	$.getJSON(appUrl + '/module/displays_info/get_data/' + serialNumber, function(data){
+		// Set count of displays
+		$('#displays-cnt').text(data.length);
+		var skipThese = ['id', 'serial_number', 'model'];
+		$.each(data, function(i,d){
+			
+			// Generate rows from data
+			var rows = ''
+			for (var prop in d){
+				// Skip skipThese
+				if(skipThese.indexOf(prop) == -1){
+                    if(prop == 'type' && d[prop] == 1){
+					   rows = rows + '<tr><th>'+i18n.t('displays.'+prop)+'</th><td>'+i18n.t('displays.internal')+'</td></tr>';
```

Comment:
I think we need to store the translation table somewhere in js so that both the listing and the client tab can read from the same location